odl/contrib/mrc/README.md

```diff
@@ -0,0 +1,21 @@
+# mrc
+
+- Data I/O functionality for the [MRC2014](http://www.ccpem.ac.uk/mrc_format/mrc2014.php) file format [Che+2015] including [FEI extended headers](http://www.2dx.unibas.ch/documentation/mrc-software/fei-extended-mrc-format-not-used-by-2dx)
+- Support for related uncompressed binary file formats with fixed-size header
+
+## Example usage
```

Comment:
This examples says absolutely nothing as is. The hard part is clearly how to extract the data you need from the header. Any chance you can do that?

Further I'd probably prefer if this was added to the examples folder so that it is run by `pytest --contrib --examples` which ensures that it at least does *something*.